src/pages/Admin/Admin_v2.tsx

```diff
@@ -63,7 +61,7 @@ const Admin_v2: React.FC = () => {
         <S.ViewWrapper>
           {selectedVenue ? (
             // @debt Venue_v2 has different structure than AnyVenue, 1 of them should be deprecated.
-            <AdminVenueView venue={selectedVenue as Venue_v2} />
+            <AdminVenueView />
```

Comment:
looks like we don't need the `@debt` comment here anymore?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/767b8397b022e9e276ed46c3f1bd384570fa4751